### ISSUE TITLE
Unmet peer dependency "ajv@^5.0.0"

### ISSUE THREAD

**Issue by MathiasSM** [postcss/postcss-loader#334]
Dependency `schema-utils` (which I don't know what's it for) has peerDep `ajv` (which I don't get why would be needed here). 

### `Error (Logs|Stacks)`
---

yarn warning message:

```
warning "postcss-loader > schema-utils > ajv-keywords@2.1.1" has unmet peer dependency "ajv@^5.0.0".
```

### `Reproduction [Code]`
---

```bash
$ yarn add --dev postcss-loader
```

### `Environment`
---

|OS|node|npm|PostCSS Loader|
|:-:|:--:|:-:|:------:|
|macOS 10.12.6|9.4.0|5.6.0|2.1.0|

**Comment by michael-ciniawsky** [comment]
Are you using `>= webpack v4.0.0-beta.0` ?

**Comment by MathiasSM** [comment]
Nope. I'm using Webpack 3.11.0

**Comment by michael-ciniawsky** [comment]
https://github.com/webpack/webpack/pull/6430 `webpack =< v4.0.0` (`>= v3.11.0`) also updated `ajv`. Good to know :) I will release a `patch` for `schema-utils` (the dependency updates are already on `master`) and the warning should be fixed. mom...

**Comment by michael-ciniawsky** [comment]
https://github.com/webpack-contrib/schema-utils/releases/tag/v0.4.4

**Comment by backwardok** [comment]
Can the upgrade to ajv happen in a major release and a new patch release be put out that downgrades ajv again? This is resulting in a number of dependency issues, one of which is eslint: https://github.com/eslint/eslint/blob/master/package.json#L37.

Since the ajv bump is a major bump, it is causing a ripple effect in all the dependencies that have it pegged to all minor releases. And, as this was included in webpack 3.11, which should have been a minor release, it is causing a lot of issues with the dependencies.
